File: app/src/main/java/com/saienko/androidthings/barman/ui/settings/adapter/CocktailElementViewHolder.java
package com.saienko.androidthings.barman.ui.settings.adapter;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageButton;
import android.widget.TextView;
import com.saienko.androidthings.barman.R;

/**
 * Created
 * User: Vasiliy Saienko
 * Date: 9/9/17
 * Time: 17:15
 */

class CocktailElementViewHolder extends RecyclerView.ViewHolder {
    TextView    textName;
    TextView    textVolume;
    ImageButton btnDelete;

    CocktailElementViewHolder(View v) {
        super(v);
        textName = v.findViewById(R.id.itemName);
        textVolume = v.findViewById(R.id.itemVolume);
        btnDelete = v.findViewById(R.id.btnDelete);
    }
}

File: app/src/main/java/com/saienko/androidthings/barman/ui/adapter/view/CocktailElementViewHolder.kt
package com.saienko.androidthings.barman.ui.adapter.view

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView
import com.saienko.androidthings.barman.R
import com.saienko.androidthings.barman.db.entity.CocktailElement

/**
 * Created
 * User: Vasiliy Saienko
 * Date: 11/10/17
 * Time: 00:15
 */
class CocktailElementViewHolder(v: View) : RecyclerView.ViewHolder(v) {

    var name: TextView = v.findViewById(R.id.itemName)
    var volume: TextView = v.findViewById(R.id.itemVolume)
    lateinit var cocktailElement: CocktailElement
}
